operator/informer_kubernetes.go

```diff
@@ -70,9 +94,15 @@ func NewKubernetesBasedInformer(sch resource.Kind, client ListWatchClient, optio
 		healthCheckName = fmt.Sprintf("%s?%s", healthCheckName, strings.Join(params, "&"))
 	}
 
+	errorHandler := DefaultErrorHandler
+	if options.ErrorHandler != nil {
+		errorHandler = DefaultErrorHandler
```

Comment:
This should be `options.ErrorHandler` 😅 

```suggestion
		errorHandler = options.ErrorHandler
```

Reply:
You didn't see anything 🙈